docs/changelog.rst

```diff
@@ -83,6 +87,11 @@ Gapless
 - Tests have been updated to always use a core actor so async state changes
   don't trip us up.
 
+- Seek events are now triggered when the seek completes. Previously the event
+  was emitted when the seek was requested, not when it completed. Further
+  changes have been made to make seek work correctly for gapless related corner
+  cases. (Fixes: :issue:`1305` PR: :pr:`1346`)
```

Comment:
"Same comments as above" means that `:pr:` must be replaced with `:issue:` here as well. ;-)